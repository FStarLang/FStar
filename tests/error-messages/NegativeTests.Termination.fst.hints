[
  "∑ò\u0003–êNázxı;Ω'O\u0018+",
  [
    [
      "NegativeTests.Termination.sumto",
      1,
      2,
      1,
      [
        "@MaxIFuel_assumption", "@query",
        "Prims_pretyping_ae567c2fb75be05905677af440075565",
        "binder_x_bb4e1c9af0265270f8e7a5f250f730e2_0", "equation_Prims.nat",
        "function_token_typing_Prims.__cache_version_number__",
        "int_inversion", "primitive_Prims.op_Addition",
        "primitive_Prims.op_Equality", "primitive_Prims.op_Subtraction",
        "projection_inverse_BoxInt_proj_0",
        "refinement_interpretation_Tm_refine_542f9d4f129664613f2483a6c88bc7c2",
        "well-founded-ordering-on-nat"
      ],
      0,
      "1174d022773ecce71cb256f1adc8b502"
    ],
    [
      "NegativeTests.Termination.snat",
      1,
      2,
      1,
      [ "@query" ],
      0,
      "c1b87ba52ceb628005038f783103307e"
    ],
    [
      "NegativeTests.Termination.__proj__S__item___0",
      1,
      2,
      1,
      [
        "@MaxIFuel_assumption", "@query",
        "refinement_interpretation_Tm_refine_58d69b53f68a55a0902003d29e7d86f7"
      ],
      0,
      "636d41fb18a705d0d126c0a258eaa210"
    ]
  ]
]