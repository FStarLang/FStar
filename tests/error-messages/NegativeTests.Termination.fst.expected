>> Got issues: [
NegativeTests.Termination.fst(21,28-21,29): (Error 19) Could not prove termination of this recursive call; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also NegativeTests.Termination.fst(18,12-21,29))
>>]
>> Got issues: [
NegativeTests.Termination.fst(28,26-28,35): (Error 19) Could not prove termination of this recursive call; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also NegativeTests.Termination.fst(25,23-28,35))
>>]
>> Got issues: [
NegativeTests.Termination.fst(35,43-35,44): (Error 19) Could not prove termination of this recursive call; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also NegativeTests.Termination.fst(31,19-36,54))
>>]
>> Got issues: [
NegativeTests.Termination.fst(42,28-42,29): (Error 19) Could not prove termination of this recursive call; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also NegativeTests.Termination.fst(40,23-42,29))
>>]
>> Got issues: [
NegativeTests.Termination.fst(55,15-55,29): (Error 19) Could not prove termination of this recursive call; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also NegativeTests.Termination.fst(53,2-55,29))
>>]
>> Got issues: [
NegativeTests.Termination.fst(67,19-67,29): (Error 19) Could not prove termination of this recursive call; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also NegativeTests.Termination.fst(64,2-67,29))
>>]
>> Got issues: [
NegativeTests.Termination.fst(75,34-75,35): (Error 19) Could not prove termination of this recursive call; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also NegativeTests.Termination.fst(71,13-75,35))
>>]
>> Got issues: [
NegativeTests.Termination.fst(80,2-82,14): (Error 19) Patterns are incomplete; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel
>>]
>> Got issues: [
NegativeTests.Termination.fst(89,29-89,31): (Error 19) Could not prove termination of this recursive call; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also NegativeTests.Termination.fst(86,15-89,31))
>>]
>> Got issues: [
NegativeTests.Termination.fst(96,20-96,26): (Error 19) Could not prove termination of this recursive call; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also NegativeTests.Termination.fst(94,2-96,26))
>>]
NegativeTests.Termination.fst(18,4-18,9): (Warning 240) Admitting NegativeTests.Termination.bug15 without a definition
NegativeTests.Termination.fst(23,4-23,18): (Warning 240) Admitting NegativeTests.Termination.repeat_diverge without a definition
NegativeTests.Termination.fst(31,4-31,17): (Warning 240) Admitting NegativeTests.Termination.ackermann_bad without a definition
NegativeTests.Termination.fst(38,4-38,14): (Warning 240) Admitting NegativeTests.Termination.length_bad without a definition
NegativeTests.Termination.fst(50,4-50,18): (Warning 240) Admitting NegativeTests.Termination.strangeZeroBad without a definition
NegativeTests.Termination.fst(61,4-61,6): (Warning 240) Admitting NegativeTests.Termination.t1 without a definition
NegativeTests.Termination.fst(69,4-69,8): (Warning 240) Admitting NegativeTests.Termination.plus without a definition
NegativeTests.Termination.fst(77,4-77,9): (Warning 240) Admitting NegativeTests.Termination.plus' without a definition
NegativeTests.Termination.fst(84,4-84,9): (Warning 240) Admitting NegativeTests.Termination.minus without a definition
NegativeTests.Termination.fst(91,4-91,7): (Warning 240) Admitting NegativeTests.Termination.xxx without a definition
Verified module: NegativeTests.Termination
All verification conditions discharged successfully
